person.rb

```diff
@@ -1,25 +1,58 @@
 require_relative 'nameable'
+
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(id, age = 0, name: 'Unknown', parent_permission: true)
     super(id)
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def can_use_services?
     of_age? || @parent_permission
   end
 
+  def rent_book(book_title, book_author, date)
+    book = Book.new(book_title, book_author)
+    rental = Rental.new(date, self, book)
+    @rentals << rental
+    rental
+  end
```

Comment:
- [ ] The method for adding a book in the Person class should look a little different, please have a look at [this article ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)for more information